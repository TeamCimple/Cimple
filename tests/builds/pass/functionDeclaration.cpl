int test()
{
	return 0;
}

int main(int argc)
{
	return test();
}

