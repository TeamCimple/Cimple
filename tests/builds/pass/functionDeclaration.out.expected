FuncDecl(Name: Var(main) ReturnType: int Parameters: PARAM(int Var(argc))  Body: CompoundStatement(Declarations:  StatementList: RETURN(Call(Receiver()FunctionName: test Params: )))) , FuncDecl(Name: Var(test) ReturnType: int Parameters:  Body: CompoundStatement(Declarations:  StatementList: RETURN(Int(0))))
